lib/web_ui/test/canvaskit/canvas_golden_test.dart

```diff
@@ -846,12 +852,14 @@ Future<void> testSampleText(String language, String text,
     paragraphHeight = paragraph.height;
     return recorder.endRecording();
   });
-  await matchPictureGolden(
-    'canvaskit_sample_text_$language.png',
-    picture,
-    region: ui.Rect.fromLTRB(0, 0, testWidth, paragraphHeight + 20),
-    write: write,
-  );
+  if (!kIssue86432Exists) {
```

Comment:
Should this be `if (kIssue86432Exists)`?

Reply:
Ah, no this is right, sorry.

Reply:
You and me both! I actually did type `if (kIssue86432Exists)` initially, then thought about it and amended the commit, lol :)